test/unit/remote_tests.rb

```diff
@@ -8,14 +8,16 @@ module Dk::Remote
   class UnitTests < Assert::Context
     desc "Dk::Remote"
     setup do
-      @hosts    = Factory.hosts
-      @ssh_args = Factory.string
-      @cmd_str  = Factory.string
+      @hosts         = Factory.hosts
+      @ssh_args      = Factory.string
+      @host_ssh_args = { Factory.string => Factory.string }
```

Comment:
@kellyredding - You might do `{ @hosts.sample => Factory.string }` to verify that it actually pulls values out of the hash for one of the hosts. Otherwise, `host_args[host]` is always returning `nil` which could return false positives.